### PR TITLE
Add pest tool

### PR DESCRIPTION
**Pull request by benjy8001**

### CODE REVIEW

File: resources/test.json [hunk 1]
```diff
@@ -54,6 +54,20 @@
             "test": "paratest --version",
             "tags": ["test"]
         },
+        {
+            "name": "pest",
+            "summary": "an elegant PHP testing Framework with a focus on simplicity, meticulously designed to bring back the joy of testing in PHP",
+            "website": "https://github.com/pestphp/pest",
+            "command": {
+                "composer-bin-plugin": {
+                    "package": "laravel/pest",
```
Comment (jakzal) [inline resources/test.json:63]: There's no composer package of this name: https://packagist.org/search/?q=laravel/pest&reason=package_not_found

File: composer.json [hunk 2]
```diff
@@ -37,7 +37,8 @@
     },
     "config": {
         "allow-plugins": {
-            "infection/extension-installer": true
+            "infection/extension-installer": true,
+            "pestphp/pest-plugin": true
```
Comment (jakzal) [inline composer.json:41]: This is the wrong place to do it as it changes the configuration for toolbox, not for its runtime.

For the correct way see examples for other tools, like phpcs:

https://github.com/jakzal/toolbox/blob/1ef9e2f185afb5a208f3af214a5b77459e68c2a6/resources/phpcs.json#L22C1-L24

File: resources/test.json [hunk 3]
```diff
@@ -54,6 +54,20 @@
             "test": "paratest --version",
             "tags": ["test"]
         },
+        {
+            "name": "pest",
+            "summary": "an elegant PHP testing Framework with a focus on simplicity, meticulously designed to bring back the joy of testing in PHP",
```
Comment (jakzal) [inline resources/test.json:59]: Description is a bit long. Can you shorten it to "an elegant PHP testing Framework with a focus on simplicity" ?

**Comment by jakzal** [comment]
There's something wrong with builds on PRs submitted by anyone but me... I think it has something to do with tokens.

Anyway, can you test it works locally with `make test-integration`?

File: composer.json (hunk 2 above)
Comment (benjy8001) [reply composer.json]: Thanks

**Comment by benjy8001** [comment]
``> There's something wrong with builds on PRs submitted by anyone but me... I think it has something to do with tokens.
> 
> Anyway, can you test it works locally with `make test-integration`?

After running make build, i've got that : 

```
phpdbg -qrr ./vendor/bin/infection --no-interaction --formatter=progress --min-msi=100 --min-covered-msi=100 --only-covered --ansi
make: execvp: phpdbg: Input/output error
make: *** [Makefile:53: infection] Error 127
```

```
╰ make test-integration
make: *** No rule to make target 'build/toolbox.phar', needed by 'test-integration'.  Stop.
```

**Comment by jakzal** [comment]
Seems to be a problem with phpdbg on your machine. In that case, you might try to run it in one of the latest jakzal/phpqa docker images.

File: resources/test.json [hunk 4]
```diff
@@ -54,6 +54,23 @@
             "test": "paratest --version",
             "tags": ["test"]
         },
+        {
+            "name": "pest",
+            "summary": "The elegant PHP Testing Framework",
+            "website": "https://github.com/pestphp/pest",
+            "sh": {
```
Comment (jakzal) [inline resources/test.json:61]: Something didn't go right here. Both `sh` and `composer-bin-plugin` should be nested inside `command`. Take a look at the earlier example I referred to: https://github.com/jakzal/toolbox/blob/1ef9e2f185afb5a208f3af214a5b77459e68c2a6/resources/phpcs.json#L21C1-L29

**Comment by jakzal** [comment]
Sorry, I just remembered you also need to build the phar, so the command should be : 

```bash
make package test-integration
```

**Comment by jakzal** [comment]
Let's try again. Thank you @benjy8001 🍺

**Comment by benjy8001** [comment]
Thanks, and sorry for the review work 🙂.

**Pull request merged**